test/cypress/utils/verify-location.js

```diff
@@ -0,0 +1,10 @@
+import locations from '../../../lib/locations'
+
+export function verifyLocation(sdk, location) {
+  expect(sdk.location.is(location)).to.equal(true)
+  let invalidLocations = Object.values(locations)
+  invalidLocations = invalidLocations.filter(item => item !== location)
+  invalidLocations.push('test-value')
+  console.log(invalidLocations)
+  invalidLocations.every(item => expect(sdk.location.is(item)).to.equal(false))
```

Comment:
Minor, but I think a `forEach` would make more sense here as we don't use the return value of this code.
```suggestion
  invalidLocations.forEach(item => expect(sdk.location.is(item)).to.equal(false))
```